pkg/platform/platform_test.go

```diff
@@ -17,6 +17,8 @@ package platform_test
 import (
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/hcl/v2"
 	"github.com/kinvolk/lokomotive/pkg/platform"
```

Comment:
Local module imports should be separate from 3rd party.
```suggestion
	"github.com/google/go-cmp/cmp"
	"github.com/hashicorp/hcl/v2"

	"github.com/kinvolk/lokomotive/pkg/platform"
```